### PR TITLE
Re-do proper Project Gutenberg extraction

### PR DESCRIPTION
**Pull request by lissyx**
for f in $(wc -l data/gutenberg/*.txt | sort -nr | head -n 120 | grep txt | awk '{ print $2 }');
do
	F=$(basename `echo $f`);
	T="gutenberg_$F";
	tail -n 100 $f | head -n 40 > $T;
done

### CODE REVIEW

**Comment by lissyx** [comment]
@mikehenrty This should also be good to merge

**Comment by mikehenrty** [comment]
awesome work @lissyx !

I still see some special characters like `--`, and sometimes there are two sentences together, like:
`Est-ce tout, monsieur? --Expliquez-moi la Trinité dit Bouvard.` Can you fix it Alex?

**Comment by lissyx** [comment]
@mikehenrty Some books were improperly regenerated, it should be better now :-)

File: server/data/fr/gutenberg_27566.txt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+Seule, Aurélie haletait de bonheur.
+Le fils ralentit sa fougue d'amour.
+Il voyait le vieillard lui sourire, surpris et fier de sa descendance.
+Ensuite ils restèrent silencieux.
+Dans la pièce obscure, ils s'entrevoyaient à peine.
+La bise avait rabattu les volets.
+Pensive, elle ramenait sur elle les plis de la robe écossaise.
+Vous étiez beaux.
+Oh ! murmura Virginie, j'ai eu toute ta force en moi, Bernard.
+Ta force qui tue et qui aime..
```
Comment (mikehenrty) [inline server/data/fr/gutenberg_27566.txt]: double period

File: server/data/fr/gutenberg_33738.txt [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+Quelle sorcière ? «Au troisième acte, nous sommes en plein vent.
+mille huit cent trente et un.
+Chez Giraldon Bovinet, un vol.
+in-8º, qui fut mis en vente vers la fin de l'année mille huit cent trente.
+Voir : FÉLIX ARVERS, par Charles Glinel.
+Reims, OEuvres Posthumes, in-douze, p.
+deux cent quarante et un.
+MES HEURES PERDUES par Félix Arvers.
+Paris, Fournier, mille huit cent trente-trois.
+puis un autre…..
```
Comment (mikehenrty) [inline server/data/fr/gutenberg_33738.txt]: strange ellipses

File: server/data/fr/gutenberg_33738.txt [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+on se battait toujours ; Et de faim ni de soif, nul ne sentait l'envie.
+Deux jours !…..
```
Comment (mikehenrty) [inline server/data/fr/gutenberg_33738.txt]: strange ellipses

File: server/data/fr/gutenberg_27566.txt (hunk 1 above)
Comment (lissyx) [reply server/data/fr/gutenberg_27566.txt]: This one comes from the book :(

File: server/data/fr/gutenberg_33738.txt (hunk 3 above)
Comment (lissyx) [reply server/data/fr/gutenberg_33738.txt]: Also comes from the original data: http://www.gutenberg.org/cache/epub/33738/pg33738.txt, with `.....`

File: server/data/fr/gutenberg_12829.txt [hunk 4]
```diff
@@ -0,0 +1,140 @@
+Les idées, essences, types, formes, principes, sont éternelles et immuables.
+[Note quatre cent vingt-six : Cette doctrine est partout dans Platon.
+l'Essai sur la Métaphysique d'Aristote, par Monsieur Ravaisson, IIIe part., l.
+deux cent quatre-vingt-onze-trois cent cinq et l'Hist.
+de la philosophie, de Ritter, l.
+deux de la trad., p.
+Il n'y a de parfaitement réel que l'individuel.
+[Note quatre cent vingt-sept : Cat., cinq., Analyt.
+logic., sept, deux cent quarante-six.) [Grec : Ou tina ta koiva.] (Simpl.
```
Comment (mikehenrty) [inline server/data/fr/gutenberg_12829.txt]: this particular text looks like it has a few problems, like `[` and splitting on abbreviations

Comment (lissyx) [reply server/data/fr/gutenberg_12829.txt]: let's remove it

**Comment by lissyx** [comment]
@mikehenrty Should be better now, but tell me if you spot anything weird, my etes are exploding now :)

**Comment by mikehenrty** [comment]
I still see a bunch of `[` and `]` characters. I think we should just remove those lines as they are usually not very conversational sentences anyway.

**Comment by lissyx** [comment]
Grrr, I'm regenerating without those lines ...

**Comment by mikehenrty** [comment]
thanks @lissyx !

**Pull request merged**